packaging/macos/sign_notarize_staple.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+[ -z "${APPLE_ID_USERNAME}" ] && echo "Please set the $APPLE_ID_USERNAME env var." >&2 && exit 1
+[ -z "${APPLE_ID_PASSWORD}" ] && echo "Please set the $APPLE_ID_PASSWORD env var." >&2 && exit 1
+[ -z "${ASC_PROVIDER}" ] && echo "Please set the $ASC_PROVIDER env var." >&2 && exit 1
+[ -z "${DMG_FILE}" ] && echo "Please set the $DMG_FILE env var." >&2 && exit 1
+
+# sign DMG
+codesign --verbose=4 --options runtime \
+    --sign "${APPLE_CODESIGN_IDENTITY}" "$(dirname $0)/entitlements.plist" $DMG_FILE
+
+# notorize
+xcrun altool --notarize-app --primary-bundle-id org.mixxx.mixxx --username $APPLE_ID_USERNAME \
+    --password $APPLE_ID_PASSWORD --asc-provider $ASC_PROVIDER --file $DMG_FILE \
+    --output-format xml > notorize_result.plist
+UUID="$(PlistBuddy -c 'Print notarization-upload:RequestUUID' notarize_result.plist)"
+
+# wait for confirmation that notorization finished
+for (( ; ; )); do
+    xcrun altool --notarization-info "$UUID" --username $APPLE_ID_USERNAME \
+    --output-format xml > notarize_status.plist
+
+    if [ "$?" != "0" ]; then
+        echo "Notarization failed:"
+        cat notarize_status.plist
+        curl "$(PlistBuddy -c 'Print notarization-info:LogFileURL' notarize_status.plist)"
+        exit 1
+    fi
+
+    NOTARIZATION_STATUS="$(PlistBuddy -c 'Print notarization-info:Status' notarize_status.plist)"
+    if [ "${NOTARIZATION_STATUS}" == "in progress" ]; then
+        echo "Waiting another 10 seconds for notarization to complete"
+        sleep 10
+    elif [ "${NOTARIZATION_STATUS}" == "success" ]; then
+        echo "Notorization succeeded"
+        break
+    else
+        echo "Notorization status: ${NOTARIZATION_STATUS}"
+    fi
+done
+
+# staple
+for (( ; ; )); do
```

Comment:
```suggestion
while true; do
```
more readable IMHO, but you decide.